packages/next/next-server/server/render.tsx

```diff
@@ -148,13 +152,23 @@ type RenderOpts = {
   App: AppType
   ErrorDebug?: React.ComponentType<{ error: Error }>
   ampValidator?: (html: string, pathname: string) => Promise<void>
+  isDataReq: boolean
+  params: { [key: string]: string }
   unstable_getStaticProps?: (params: {
-    params: any | undefined
+    params: { [key: string]: string | string[] }
```

Comment:
Add these types in a new PR? 🙏

Reply:
Opened PR here: https://github.com/zeit/next.js/pull/10259